bin/adhoc-sign.py

```diff
@@ -14,37 +14,40 @@ def main(args: argparse.Namespace):
         "macOS": "https://github.com/mozilla/mozregression"
         f"/releases/download/{release}/{filename}",
         "windows": "https://github.com/mozilla/mozregression"
-        f"/releases/download/{release}/mozregression-gui.exe",
+        f"/releases/download/{release}/mozregression-gui-unsigned.exe",
     }
 
     operating_systems = {
         "macOS": ["macapp"],
+        "windows": ["autograph_authenticode_sha2"],
     }
 
     params = {}
 
-    for os, signing_formats in operating_systems.items():
-        url = urls[os]
-        response = requests.get(url)
-        if response.status_code != 200:
-            raise ValueError(f"Could not fetch {url} ({response.status_code})")
+    os = args.os
```

Comment:
Why is the loop removed?

Reply:
In practice, this has to be done separately for each OS in different jobs. When I first implemented this I wasn't sure if one manifest can be used or not.